guides/source/working_with_javascript_in_rails.md

```diff
@@ -146,6 +146,53 @@ concatenator. We can serve our entire JavaScript bundle on every page, which
 means that it'll get downloaded on the first page load and then be cached on
 every page after that. Lots of little benefits really add up.
 
+As usual, Rails has opinions about how the software should be structured and this applies to JavaScript too. Rails provides an `app\javascript` directory as a home for our JavaScript. If we use more and more JavaScript in our app, it is worth spreading out the files across a directory structure, but for now, it is better to split the JavaScript and put it in two places. The function itself should go in `app\javascript\paint.js` after adding the export statements
```

Comment:
Paths are commonly written with forward slashes in the documentation.

```suggestion
As usual, Rails has opinions about how the software should be structured and this applies to JavaScript too. Rails provides an `app/javascript` directory as a home for our JavaScript. If we use more and more JavaScript in our app, it is worth spreading out the files across a directory structure, but for now, it is better to split the JavaScript and put it in two places. The function itself should go in `app/javascript/paint.js` after adding the export statements.
```

Reply:
Perhaps the example could point towards a file inside an `app/javascript/components/` directory?

This could help structure the first JavaScript files in the app and simplify this text down to something like this:

```suggestion
As usual, Rails has opinions about how software should be structured and this applies to JavaScript too. The function would go for example in `app/javascript/components/paint.js`, with the following export statement.
```